web/packages/core/test/config.ts

```diff
@@ -0,0 +1,30 @@
+import { strict as assert } from "assert";
+import { parseColor, parseDuration } from "../src/internal/builder";
+
+describe("Color parsing", function () {
+    it("should parse a valid RRGGBB hex, with hash", function () {
+        assert.strictEqual(parseColor("#A1B2C3"), 0xa1b2c3);
+    });
+
+    it("should parse a valid RRGGBB hex, without hash", function () {
+        assert.strictEqual(parseColor("#1A2B3C"), 0x1a2b3c);
```

Comment:
This should be testing without the hash `#`?

Reply:
Nice catch!